app/models/metric.rb

```diff
@@ -218,6 +218,10 @@ def disabled_for_plan?(plan)
     !enabled_for_plan?(plan)
   end
 
+  def service?
+    owner_type == 'Service'
```

Comment:
```suggestion
    service_id || owner_type == 'Service'
```

I'm not sure whether metrics can have a `service_id` but no owner. I wanted in the past to make metrics always use `owner` instead of the obsoleted `service` but I'm not sure that I came to finalize that.

Reply:
Actually for consistency, better add a method `service_metric?` to `app/lib/backend_api_logic/metric_extension.rb`. See `#backend_api_metric?`.

Reply:
Agree with @akostadinov 
in fact, I think the following can be used in the view:
```
@metrics.find { |metric| metric.hits? && !metric.backend_api_metric? }.system_name
```
Or, add a `service_metric?` method in that `metric_extension.rb`

Reply:
@akostadinov @mayorova Changes are done